conda

```diff
@@ -13,10 +13,28 @@ conda create --name 'ENVNAME' python=3.10
 # To install from requirements.txt
 conda install --file requirements.txt
 
+# To clone an environment
+conda create --clone 'ENVNAME' -n 'NEWENV'
+
 # To activate a virtual environment
 conda activate 'ENVNAME'
 
+# To deactivate a virtual environment
+conda deactivate
+
 # To remove and environment
 conda env remove --name 'ENVNAME'
 # or
 conda env remove --prefix /path/to/env
```

Comment:
```suggestion
conda env remove --prefix <path/to/env>
```

For frequently used placeholders like paths we can omit quotes I guess like we do in TlDr project.